test/integration/targets/docker_image/tasks/tests/options.yml

```diff
@@ -0,0 +1,181 @@
+---
+- name: Registering image name
+  set_fact:
+    iname: "{{ name_prefix ~ '-options' }}"
+    iname_1: "{{ name_prefix ~ '-options-1' }}"
+
+- name: Registering image name
+  set_fact:
+    inames: "{{ inames }} + [iname, iname_1]"
+
+####################################################################
+## buildargs #######################################################
+####################################################################
+
+- name: buildargs
+  docker_image:
+    path: "{{ role_path }}/files"
+    name: "{{ iname }}"
+    buildargs:
+      TEST1: val1
+      TEST2: val2
+      TEST3: "True"
+  register: buildargs_1
+
+- name: buildargs (idempotency)
+  docker_image:
+    path: "{{ role_path }}/files"
+    name: "{{ iname }}"
+    buildargs:
+      TEST1: val1
+      TEST2: val2
+      TEST3: "True"
+  register: buildargs_2
+
+- name: cleanup
+  docker_image:
+    name: "{{ iname }}"
+    state: absent
+
+- assert:
+    that:
+    - buildargs_1 is changed
+    - buildargs_2 is not changed
+  when: docker_py_version is version('1.6.0', '>=')
+
+- assert:
+    that:
+    - buildargs_1 is failed
+    - buildargs_2 is failed
+  when: docker_py_version is version('1.6.0', '<')
+
+####################################################################
+## container_limits ################################################
+####################################################################
+
+- name: container_limits
+  docker_image:
+    path: "{{ role_path }}/files"
+    name: "{{ iname }}"
+    container_limits:
+      memory: 5000000
+      memswap: 7000000
+  register: container_limits_1
+
+- name: cleanup
+  docker_image:
+    name: "{{ iname }}"
+    state: absent
+
+- name: container_limits (Failed due to min memory)
+  docker_image:
```

Comment:
You need a `ignore_errors: yes` here, otherwise the tests will stop because this task fails (same for the one below).

Reply:
Resolved.